robottelo/ui/base.py

```diff
@@ -415,6 +415,14 @@ def edit_entity(self, edit_loc, edit_text_loc, entity_value, save_loc):
         self.text_field_update(edit_text_loc, entity_value)
         self.click(save_loc)
 
+    def set_limit(self, limit):
+        """Specify content host limit value for host collection or activation
+        key entities.
+        """
+        self.click(locators['hostcollection.usage_limit_checkbox'])
+        if limit != 'Unlimited':
+            self.field_update('hostcollection.usage_limit', limit)
+
```

Comment:
I dont think we add entity specific properties in base.  I think these should be added to individual entities.